src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/CopyFileTests.cs

```diff
@@ -64,9 +64,9 @@ public void CopyClosedFile_ThrowsInvalidOperationException()
             }
         }
 
-        [ActiveIssue(25665)]
         [Fact]
-        public void CopyFile_RaisesInvalidPath()
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)] 
+        public void CopyFile_RaisesInvalidPath_Desktop()
```

Comment:
Just to confirm, all of these changes to behavior are going to make their way back to netfx?

Reply:
@JeremyKuhne  can you ans this question

Reply:
Eventually, under quirk.